### DISCUSSION TITLE
「より多くの方が意見を表明できる」について

### DISCUSSION THREAD

**Discussion by blono** [increments/qiita-discussions#3] [ヘルプ] [answered]
ユーザーが意見を表明しないことの一つとして、投稿した意見が本当に運営に把握され、サービスの更新につながるのかどうか疑問に感じている人が多い、さらに運営に報告をすること自体が手間であるの相乗効果があると思います。
Qiita に限らず、愚痴といったらあれですが、それを運営に報告せずに、Twitter に書いて終わりという方も多くいらっしゃいますが、そのようなことになるのは運営に意見を言っても本当に対応しているのか不透明すぎて信頼できない、というのがあると思います。
実際、同じ意見を運営に報告し続けているのに、一向に改善されないから、もう報告するのをやめた、等です。

そこでまず、株式会社の機関設計として、

1. 取締役（会）は、業務執行につきその適法性のみならず「妥当性」についても責任を追うこと
2. 取締役は、それについて任務懈怠責任を負うこと
3. 監査役は取締役に対して適法性の監査を行うこと
4. 取締役はユーザーからの意見に対してちゃんと対応しているのかどうか、部門長へヒアリングや調査を行っていること（妥当性検査）
5. 部門長は取締役に怒られたくないので、しっかりとユーザーの意見を反映させる体制を築き上げていること
6. 現場の人間は部門長に怒られたくないので、部門長が決めた体制をしっかりと守っていること

このような株式会社の機関設計があるため
「ユーザー様の意見に対しては、取締役や監査等で1件1件大切に、適切に対応を行っている」ことを質問フォーム内に記載する。
また、「ユーザー様の意見について、どのような対応を行ったのかを回答する」ことも重要だと思います。
例えば、新規の意見であったのであれば、issue として登録したとか。
既出であれば、issue に登録済みであり、今回の要望を受けて priority を変更したとか。
という具体的な解答が運営から帰ってくるようになれば、面倒くさくても意見を言う人が増えるのではないかと思いました。

**Comment by uhooi** [comment]
もちろん Qiita 以外のサービスにもいえますが「お問い合わせが不透明で信頼できない」のは私もユーザーとして思います。
Twitter  などの SNS に投稿して共感（いいねや RT）を多数得られれば「世論」となり、運営が対応せざるを得なくなるので、よりお問い合わせのような機能は無意味と感じている人は多いと思っています。

Twitter は悪い意見が目立つので、GitHub にパブリックな議論の場を用意するのは有用な気がしています。
成功することを祈っています！

**Comment by kskwtnk** [comment, marked as answer]
@blono 
ご意見ありがとうございます！

仰るとおり、ユーザーのみなさまからいただいた意見について、これまでは十分に対応・反応ができていませんでした。
我々としても課題に感じており、Discussionsを使って改善していきたいと考え、今回クローズドベータテストを実施するに至っています。

いただいた意見に対して返信をしたり、対応完了した場合もアナウンスをしたり、今後はそういった動きをしていこうと思っています！

また、代表や事業責任者が集まる会議を毎週開いており、実は今日も開催されていました。
今日時点で作成されたDiscussionは全て読ませていただいて、とても貴重な意見だから今後は是非参考にしていきましょうと話しました。

しっかりと返信やアナウンスをして、みなさんから意見をいただき、より良い改善ができるサイクルをまわせるようにしていきます！

**Comment by kskwtnk** [comment]
@uhooi 

ありがとうございます！

パブリックな場を用意することで、ユーザーさま同士にとっても、我々運営にとっても良い開発ができれば良いなと考えています

> 成功することを祈っています！

とても嬉しいお言葉です！頑張ります！
